src/interface/whitelist/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
```

Comment:
Here you should add the is whitelist route as well so that same user can check if the address is whitelisted already or not to confirm. Will be important if we do any automation setup.